### ISSUE TITLE
Readme PDF blank on page 9

### ISSUE THREAD

**Issue by Osspial** [KhronosGroup/Vulkan-Docs#203]
When attempting to open `readme.pdf` in `./src/spec`, page 9 is blank with PDF-xChange giving the error `Error [PDF Structure 40]: Invalid reference table (xref)`. The error persists when attempting to build a pdf from the provided `readme.tex` file. This appears to be related to the addition of `validextensionstructs` documentation.

**Comment by krOoze** [comment]
and in PDF.js based viewers (e.g. pdf preview here on GitHub). Works fine in Foxit Reader though...

**Comment by oddhack** [comment]
@TobiasHector  I think this happened when you updated the readme - could you check that your LaTeX toolchain is doing the right thing?

**Comment by TobiasHector** [comment]
It looks like someone checked in a merge conflict - the bottom of the file has this in:

```
>>>>>>> 1.0
trailer
<< /Size 650
/Root 648 0 R
/Info 649 0 R
<<<<<<< HEAD
/ID [<D56774A5F6E1307686D259DFA280E39A> <D56774A5F6E1307686D259DFA280E39A>] >>
startxref
224553
=======
/ID [<761474DAF29E3B552170784BBD125F98> <761474DAF29E3B552170784BBD125F98>] >>
startxref
224908
>>>>>>> 1.0
%%EOF
```

I've created a fix (regenerated the file) which should hit the next release. For anyone trying to read the thing, here's a fixed version:
[readme.pdf](https://github.com/KhronosGroup/Vulkan-Docs/files/248793/readme.pdf)

**Comment by oddhack** [comment]
My proposed fix is to stop checking in the PDF and generate it in the same OUTDIR as the spec, style guide etc. outputs. Actually generally speaking there should be no generated files in the tree, but we've had to leave some of them (like the asciidoc includes for prototypes, struct definitions, etc.) in as a sop to people with build environments that don't have all the required tools. Hopefully we can get away from that in the future.

**Comment by ratchetfreak** [comment]
If you consider that PDF readers have been an attack vector for malware it's not that surprising that they have a sane default behavior on malformed input.

**Comment by oddhack** [comment]
This should be fixed in the 1.0.13 spec update. The README has been moved out of the repo and is generated in the registry area (khronos.org/registry/vulkan) instead.
